examples/reference-entity-model/src/main/ml-config/security/users/hub-developer.json

```diff
@@ -5,6 +5,7 @@
   "role": [
     "data-hub-developer",
     "hub-central-user",
+    "hub-central-entity-model-writer",
```

Comment:
Note that this will be simplified soon once we have the hub-central-developer role. Will only need to add that then.

Reply:
Yeah, just added so PM can use hub-developer  has access to everything for now, when we get hub-central-developer-role, will remove this.